### DISCUSSION TITLE
Using in JavaFx(Desktop App)

### DISCUSSION THREAD

**Discussion by ezizrehimov** [mapsforge/mapsforge#1231] [Q&A] [answered]
Hi. can i view openstreetmap(offline) file using mapsforge in JavaFx(Desktop App). and can you share tutorial?

**Comment by devemux86** [comment, marked as answer]
Search the issues list and you can find #928 and #1171.

**Reply by ezizrehimov** [reply]
@devemux86 
but this is doesn't work anymore.   #1171
can you do any sample tutorial?(offline usage in JavaFx .thank you :)
